platform/android/export/export_plugin.cpp

```diff
@@ -3086,6 +3095,9 @@ Error EditorExportPlatformAndroid::export_project_helper(const Ref<EditorExportP
 
 	load_icon_refs(p_preset, main_image, foreground, background, monochrome);
 
+	const String icon_xml_path = "res/mipmap-anydpi-v26/icon.xml";
+	const String themed_icon_xml_path = "res/mipmap-anydpi-v26/themed_icon.xml";
```

Comment:
Similar comment, these constants can be moved at the top level alongside https://github.com/godotengine/godot/blob/d741a646a5ee20e77c1d0d9d4fbfed03d7b8dcc3/platform/android/export/export_plugin.cpp#L222.

This will allow to remove the additional parameter added to `_copy_icons_to_gradle_project(...)`.